cmd/bnscli/main.go

```diff
@@ -32,12 +32,13 @@ import (
 //   $ bnscli escrow-proposal -escrow 1 | bnscli sign | bnscli submit
 //
 var commands = map[string]func(input io.Reader, output io.Writer, args []string) error{
-	"transfer-proposal": cmdNewTransferProposal,
 	"escrow-proposal":   cmdNewEscrowProposal,
+	"reset-revenue":     cmdResetRevenue,
 	"sign":              cmdSignTransaction,
 	"submit":            cmdSubmitTransaction,
-	"view":              cmdTransactionView,
+	"transfer-proposal": cmdNewTransferProposal,
 	"version":           cmdVersion,
+	"view":              cmdTransactionView,
```

Comment:
I like this `view` subcommand. Super helpful and great to include in the PR description.

Reply:
Fun fact: it was your idea to add it to the `validators`. The original motivation was that you need to see what the message is before you can trust to sign it.